### PR TITLE
[ISSUE#1850] refactor ExtensionLoaderTest, different jdk version has different exception message. (#1850)

### PR DESCRIPTION
**Pull request by HessTina-YuI**
…nt exception message.(#1850)

// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [X] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor/).
- [X] You submit test cases (unit or integration tests) that back your changes.
- [X] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java [hunk 1]
```diff
@@ -183,10 +183,6 @@ public void testGetExtensionLoaderCanNotInstantiatedSPI() {
             ExtensionLoader.getExtensionLoader(JdbcSPI.class).getJoin("canNotInstantiated");
             fail();
         } catch (IllegalStateException expected) {
-            assertThat(expected.getMessage(), containsString(
```
Comment (tydhot) [inline shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java:186]: remove the test, it is meaningless to remain the code without assert

**Review by tydhot — CHANGES_REQUESTED** [review]

File: shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java (hunk 1 above)
Comment (HessTina-YuI) [reply shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java:186]: ok, I will delete

Comment (dengliming) [reply shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java:186]: I suggest we can keep this line.
`"Extension instance(name: canNotInstantiated, class: class org.apache.shenyu.spi.fixture.CanNotInstantiatedSPI)  "`

Comment (tydhot) [reply shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java:186]: agree

Comment (HessTina-YuI) [reply shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java:186]: ok, this message, the jdk8 and jdk11 are the same

**Review by dengliming — APPROVED** [review]

**Pull request merged**